tensorflow/compiler/mlir/tfrt/transforms/lmhlo_to_gpu/pattern_utils.h

```diff
@@ -19,12 +19,25 @@
 #include "mlir/IR/Types.h"
 #include "tensorflow/stream_executor/dnn.h"
 #include "tfrt/gpu/wrapper/cublas_wrapper.h"  // from @tf_runtime
+#include "tfrt/gpu/wrapper/rocblas_wrapper.h" // from @tf_runtime
```

Comment:
I will merge this change manually because the duplicate include of `rocblas_wrapper.h` trips up the copybara import.